config/sitemap.rb

```diff
@@ -0,0 +1,19 @@
+# https://github.com/kjvarga/sitemap_generator#sitemapgenerator
+SitemapGenerator::Sitemap.sitemaps_path = "sitemaps"
+SitemapGenerator::Sitemap.create_index = "auto"
+SitemapGenerator::Sitemap.compress = false
+SitemapGenerator::Sitemap.default_host = URI.parse(Seek::Config.site_base_url)
+
+SitemapGenerator::Sitemap.create do
+  Seek::Util.searchable_types.each do |type|
+    add  polymorphic_path(type), lastmod: type.maximum(:updated_at), changefreq: 'daily', priority: 0.7
+  end
+end
+
+Seek::Util.searchable_types.each do |type|
+  SitemapGenerator::Sitemap.create(filename: type.table_name, include_root: false) do
+    type.authorized_for('view', nil).find_all do |obj|
+      add polymorphic_path(obj), lastmod: type.maximum(:updated_at), changefreq: 'daily', priority: 0.7
```

Comment:
`lastmod` for the object should be its own `updated_at`: `obj.updated_at`

Reply:
Fixed in [4d68328](https://github.com/seek4science/seek/pull/1578/commits/4d68328bdbcde9a2cdf250893d9b498f8c70d9b2).